spec/controllers/boards_controller_spec.rb

```diff
@@ -27,24 +28,24 @@
     end
   end
 
-  describe "GET edit" do
-    let(:board) {create(:board)}
+  describe "GET edit" do 
+    let(:board) {create(:board, owner: user)}
     it "renders the edit template" do
       get :edit, id: board.id
       expect(response).to render_template(:edit)
     end
   end
 
   describe "DELETE destroy" do
-    let!(:board) {create(:board)}
+    let!(:board) {create(:board, owner: user)}
```

Comment:
DRY !!!! czemu to jest schowane jak prawie w kazdmy tescie to uzywasz :) wyciagaj to z tego describe !